specs/adagrams.spec.js

```diff
@@ -101,9 +101,11 @@ describe('Adagrams', () => {
 
     describe('in case of tied score', () => {
       const expectTie = (words) => {
-        const scores = words.map(Adagrams.scoreWord);
+        const scores = words.map((word) => Adagrams.scoreWord(word));
         const highScore = scores.reduce((h, s) => h < s ? s : h, 0);
```

Comment:
You shouldn't need to change the tests at all. Also, I don't think this change should make any difference in how this code behaves.